### DISCUSSION TITLE
Game crash when clicking "Update Cache" in cheat

### DISCUSSION THREAD

**Discussion by Alestarov** [YimMenu/YimMenu#1830] [Q&A] [answered]
I have a problem that I can not solve for the past week.
 When I try to inject the cheat into the game it is injecting, but when I press "Update Cache" the game freezes and crashes after a second. Inject occurs in the story game. I used different injectors: dll injector, Extreme injector. but I am currently using Xenos injector. Xenos Injector and Yim menu are downloaded from official git hub releases.
Help me please.

**Comment by Alestarov** [comment, marked as answer]
I already solved the problem completely by accident) I had fashion schedules installed and because of them I crashed

**Reply by xiaoxiao921** [reply]
What is "fashion schedules" ? Can you link that mod?

**Reply by Alestarov** [reply]
Sorry, not fashion schedules but a Graphic mods. I translated through a google translator). there is an application Network Graphics, there are collected modifications for GTA online and GTA RP (graphics mods, maps, etc.). I had mods installed from this app. Because of this, the cheat did not work.

**Comment by Tessio** [comment]
In conclusion modified RPFs will cause an "ERR_GEN_INVALID" error when trying to create the mod menu cache

**Comment by lonelybud** [comment]
for me the game freeze for a while. then unfreeze when done.
